app/assets/stylesheets/common/foundation/variables.scss

```diff
@@ -65,6 +65,13 @@ $font-down-4: var(--font-down-4) !default;
 $font-down-5: var(--font-down-5) !default;
 $font-down-6: var(--font-down-6) !default;
 
+$font-up-3-rem: var(--font-up-3-rem);
```

Comment:
Do we need to add SCSS variables at all? Can we directly use `var(--font-up-3-rem)` etc. instead? 

(I think the existing `$font-up/down` variables are there for legacy reasons, we should eventually phase them out as well.)

Reply:
I did not know why we still use SCSS variables and I did not want to break consistency. I will push a follow up commit.